### PR TITLE
3.11 Release Notes Edits

### PR DESCRIPTION
**Pull request by bmcelvee**
Edits to the 3.11 Release Notes - mostly for Registry.

### CODE REVIEW

File: release_notes/ocp_3_11_release_notes.adoc [hunk 1]
```diff
@@ -325,7 +328,12 @@ The new registry, `registry.redhat.io`, requires authentication for access to
 images and hosted content on {product-title}. Following the move to the new
 registry, the existing registry will be available for a period of time.
 
-See Accessing and Configuring the Red Hat Registry for more information.
+See xref:../../install_config/registry/index.adoc#auth-enabled-registry[Authentication Enabled Red Hat Registry]
+for more information. 
+
```
Comment (bmcelvee) [inline release_notes/ocp_3_11_release_notes.adoc]: This link won't work until https://github.com/openshift/openshift-docs/pull/11506 is merged.

**Comment by bmcelvee** [comment]
Preview build: http://file.rdu.redhat.com/~bmcelvee/01102018/rel-note-3.11/release_notes/ocp_3_11_release_notes.html

**Comment by bmcelvee** [comment]
@bparees PTAL

File: release_notes/ocp_3_11_release_notes.adoc [hunk 2]
```diff
@@ -511,7 +514,12 @@ The new registry, `registry.redhat.io`, requires authentication for access to
 images and hosted content on {product-title}. Following the move to the new
 registry, the existing registry will be available for a period of time.
 
-See Accessing and Configuring the Red Hat Registry for more information.
+See xref:../install_config/configuring_red_hat_registry.adoc#install-config-configuring-red-hat-registry[Authentication Enabled Red Hat Registry]
+for more information.
```
Comment (bparees) [inline release_notes/ocp_3_11_release_notes.adoc]: i assume this is what i'm reviewing? looks fine.

Comment (bmcelvee) [reply release_notes/ocp_3_11_release_notes.adoc]: Yeah, thanks!

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Pull request merged**